### PR TITLE
Fix for messages in read message function

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id:Tendrl/node-agent#279

Read message function takes only two arguments as
parameter but gevent is passing 3 parameters for read function.
Third is args.

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n please review

File: tendrl/node_agent/objects/cluster_message/__init__.py [hunk 1]
```diff
@@ -10,6 +10,9 @@ def __init__(self, **cluster_message):
         self.value = 'clusters/%s/Messages/%s'
         self._etcd_cls = _ClusterMessageEtcd
 
+    def load_definition(self):
```
Comment (shtripat) [inline tendrl/node_agent/objects/cluster_message/__init__.py]: Is it mandatory to have `load_definitions()` with empty implementation?

Comment (GowthamShanmugam) [inline tendrl/node_agent/objects/cluster_message/__init__.py]: @shtripat with out this message class gives error

Comment (shtripat) [inline tendrl/node_agent/objects/cluster_message/__init__.py]: ok

Comment (r0h4n) [inline tendrl/node_agent/objects/cluster_message/__init__.py]: overloading load_definitions is a way to bypass the definitions system (i.e you dont have to define the object/flow/atom in the definitions file)

**Pull request merged**